config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,4 @@
+Sidekiq.configure_server do |config|
```

Comment:
we should probably also add the monitoring interface https://github.com/mperham/sidekiq/wiki/Monitoring - same way we do for DJ: 

https://github.com/thepracticaldev/dev.to/blob/2415392da5895adcbccc4f0a3efe67c807568d19/config/routes.rb#L13-L15